src/GovUk.Education.ExploreEducationStatistics.Content.Model/DataBlockSummary.cs

```diff
@@ -13,4 +13,4 @@ public class DataBlockSummary
 
         public List<string> DataDefinitionTitle { get; set; }
     }
-}
\ No newline at end of file
+}
```

Comment:
I think we would expect to lose that data when the key stat gets removed - I don't think anyone was expecting that data to be kept?

If old key stat data does still exist in the `DataBlock_Summary` column, I think that is an existing issue of not cleaning it up, rather than something we need to keep.

I'll run it past Cam regardless.

Reply:
I've run it past Cam and he is fine with that data being lost.